src/Normalizer/PayloadNormalizer.php

```diff
@@ -12,10 +12,11 @@ public function normalize(Payload $payload): array
     {
         $result = [
             'sub' => $payload->getId(),
-            // 'exp' => $payload->
+            'exp' => $payload->getExpireAt(),
```

Comment:
would be nice to create some constant file for all reserved names of jwt that lib supports